console/console.go

```diff
@@ -82,6 +90,18 @@ func GetConsolePluginCR(consolePort int, basePath string, serviceNamespace strin
 				Port:      int32(consolePort),
 				BasePath:  basePath,
 			},
+			Proxy: []consolev1alpha1.ConsolePluginProxy{
+				{
+					Type:      consolev1alpha1.ProxyTypeService,
+					Alias:     tlsService + "-proxy",
```

Comment:
we decided alias to be `mcg-managed-prom` ??
I am okay with `mcg-prometheus-proxy` as well, but it is something used in UI so need to be sure we are on same page.

Reply:
If it's okay can we pin it to `prometheus-proxy`? If you've already made changes using the above name on your end, then LMK and I'll revert this back to `mcg-managed-prom`.

Reply:
okay... keeping it as `prometheus-proxy`... making same change in UI as well...